recipe/migrations/boost1780.yaml

```diff
@@ -1,7 +1,9 @@
 __migrator:
   build_number: 1
   kind: version
-  migration_number: 1
+  migration_number: 2
 boost:
 - 1.78.0
+boost_cpp
+- 1.78.0
 migrator_ts: 1649717600.9611828
```

Comment:
does this also need to be upaded?

Reply:
I don't know, this is my first time submitting a migration. Does it?

Reply:
i think it might be helpful. maybe. these "fixes" always confuse me

Reply:
```
python -c "import time; print(f'{time.time():.0f}')"
```

Reply:
OK updated in https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3403/commits/427c7752d071e66128e960501a1b4a4e7115101e